cfme/automate/explorer/klass.py

```diff
@@ -371,8 +371,11 @@ def row_id(self):
             max_time = Input(name=ParametrizedString('fields_max_time_{@row_id}'))
 
             def delete(self):
-                self.browser.click(
-                    './/img[@alt="Click to delete this field from schema"]', parent=self)
+                if self.browser.product_version < '5.10':
```

Comment:
Question:
Why not use VersionPick here?

Reply:
VersionPick is a descriptor and it's supposed to be used for class attributes. For view instances `self.browser.product_version` should be used.

Reply:
Yeh VersionPick could be used here, though it is generally thought to be more efficient to do a simple comparison here.